pkg/framework/operators/baseoperator.go

```diff
@@ -5,24 +5,32 @@ import (
 	"fmt"
 	"github.com/ghodss/yaml"
 	"io/ioutil"
+	appsv1 "k8s.io/api/apps/v1"
```

Comment:
Is rearanging code necessary ?

Reply:
Need to add `go fmt` to git precommit hooks  to avoid rearrangement of code ;)

Reply:
Agreed!